PubnativeLite/PubnativeLite/Ad Request/VWAdFactory.m

```diff
@@ -135,4 +135,21 @@ - (void)setIDFA:(VWAdRequestModel *)adRequestModel {
     }
 }
 
+- (void)setContentCategoryIDsForAdRequestModel:(VWAdRequestModel *)adRequestModel withContentCategoryIDs:(NSMutableArray *)contentCategoryIDs {
+    NSMutableString *contentCategoryIDsString = [NSMutableString new];
+    if (contentCategoryIDs.count) {
```

Comment:
Do you mean here to use nil ? or your intention is to check the count ?

Reply:
The reason for this is to form the c parameter correctly based on the number of content categories. The ticket itself is to add support for multiple content category IDs.

Reply:
all fine then! thanks